src/components/LoansByCategory/MFIRecommendations/MFIHero.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+	<div class="tw-w-full">
+		<div class="tw-flex tw-items-center tw-gap-1 tw-px-2 lg:tw-px-0">
+			<!-- eslint-disable max-len -->
+			<img class="tw-w-4 lg:tw-w-6" src="@/assets/images/mfi_recommendations/earth.svg" alt="mfi_recommendations">
+			<h3 class="tw-text-h3 tw-text-primary tw-whitespace-nowrap">
+				Field Partner Spotlight
+			</h3>
+			<hr class="tw-border-tertiary tw-w-full tw-ml-2 tw-hidden lg:tw-block">
+		</div>
+		<div class="tw-mt-2 lg:tw-mt-4 tw-px-2">
+			<h2 class="tw-text-h2 tw-text-primary">
+				Fundación Pro Mujer
+			</h2>
+			<h6 class="tw-text-subhead tw-text-primary">
+				15 years with Kiva, 52,908 borrowers supported
+			</h6>
+		</div>
+		<div class="tw-flex tw-flex-col lg:tw-flex-row tw-my-3">
+			<div class="md:tw-w-2/3 md:tw-px-2 tw-relative">
+				<!-- eslint-disable max-len -->
+				<img class="md:tw-rounded" src="@/assets/images/mfi_recommendations/fundacionpromujer.jpg" alt="fundacionpromujer">
```

Comment:
Should this image have our tw-rounded-sm class applied?

Reply:
There is not design for that size, but just mobile is considering no-rounded image.